question16/main.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
```

Comment:
I thought it was a good function!

Reply:
I found it from Yuka san's code and imitated it!! It is very useful!!